tests/api/test_utils.py

```diff
@@ -90,3 +92,33 @@ def test_get_lightly_server_location_from_env(self):
         os.environ["LIGHTLY_SERVER_LOCATION"] = "https://api.dev.lightly.ai/ "
         host = get_lightly_server_location_from_env()
         self.assertEqual(host, "https://api.dev.lightly.ai")
+
+    def test_paginate_endpoint(self):
```

Comment:
Could you add one more test with an empty reply?

Reply:
added